### ISSUE TITLE
Gentrification indicator data

### ISSUE THREAD

**Issue by austensen** [austensen/capstone#4]
- [x] get all variables from NCDB
- [x] create tract-level gentrification indicator
- [x] create tract -> pcsa crosswalk
- [x] aggregate tract-level variables, and calculate PCSA-level gentrification indicator
- [x] map tract and PCSA gentrifying/non-gentrifying neighbourhoods
- [x] add in other covariates
- [x] re-create analysis from last semester

**Comment by austensen** [comment]
All the gentrification indicators are ready now, at both tracts and PCSAs

**Comment by austensen** [comment]
Hey @ganzamy, I forgot to send you this earlier. For adding the other census variables, you can just use [gentrification_data_prep.R](https://github.com/austensen/capstone/blob/master/gentrification_data_prep.R). It's set up to be run at FC so you can access the J drive for NCDB data and everything.

**Comment by austensen** [comment]
Hey team, I've finished the [gentrification status by PCSA map](https://github.com/austensen/capstone/blob/master/gent_map_pcsa.png). I think it works. The 'neighbourhoods' are a little weird in that they differ wildly in size, but the map overall looks roughly like what you'd expect (compared to what it looks like at the tract-level for example). 
I took a stab at a colour scheme (since the previous one was a little harsh looking), let me know what you think in #6.
